src/crawlers/cheerio_crawler.js

```diff
@@ -171,7 +171,29 @@ const CHEERIO_OPTIMIZED_AUTOSCALED_POOL_OPTIONS = {
  *   For more information, see the [documentation](https://docs.apify.com/proxy).
  * @property {HandleFailedRequest} [handleFailedRequestFunction]
  *   A function to handle requests that failed more than `option.maxRequestRetries` times.
- *
+ * @property {Array<Hook>} [preNavigationHooks]
+ *   Async functions that are sequentially evaluated before the navigation. Good for setting additional cookies
+ *   or browser properties before navigation. The function accepts two parameters, `crawlingContext` and `gotoOptions`,
+ *   which are passed to the `requestAsBrowser()` function the crawler calls to navigate.
+ *   Example:
+ * ```
+ * preNavigationHooks: [
+ *     async (crawlingContext, gotoOptions) => {
+ *         gotoOptions.forceUrlEncoding = true;
+ *     }
+ * ]
+ * ```
+ * @property {Array<Hook>} [postNavigationHooks]
+ *   Async functions that are sequentially evaluated after the navigation. Good for checking if the navigation was successful.
+ *   The function accepts `crawlingContext` as an only parameter.
```

Comment:
**the** only parameter. And yeah, I see that it was wrong in the original comment :)